src/Http/Routing/src/Internal/MapActionExpressionTreeBuilder.cs

```diff
@@ -431,7 +431,7 @@ static async Task ExecuteAwaited(ValueTask<T> task, HttpContext httpContext)
             return ExecuteAwaited(task, httpContext);
         }
 
-        private static async ValueTask ExecuteTaskResult<T>(Task<T> task, HttpContext httpContext) where T : IResult
+        private static async Task ExecuteTaskResult<T>(Task<T> task, HttpContext httpContext) where T : IResult
```

Comment:
Should this be `ExecuteTaskResultAsync` for consistency to Async function suffix?

Reply:
Maybe, but it's a private method and the not name was changed by this PR. I wouldn't be interested in a PR that just changed that.